src/app.js

```diff
@@ -64,7 +65,15 @@
     'app.stack',
     'app.status'
   ])
-  .config(function ($compileProvider, $httpProvider, $locationProvider, $stateProvider, $uiViewScrollProvider, $urlRouterProvider, dialogsProvider, gravatarServiceProvider, RestangularProvider, BASE_URL, EXCEPTIONLESS_API_KEY, $ExceptionlessClient, stripeProvider, STRIPE_PUBLISHABLE_KEY, USE_HTML5_MODE) {
+  .config(function ($compileProvider, $httpProvider, $locationProvider, $stateProvider, $uiViewScrollProvider, $urlRouterProvider, $translateProvider, dialogsProvider, gravatarServiceProvider, RestangularProvider, BASE_URL, EXCEPTIONLESS_API_KEY, $ExceptionlessClient, stripeProvider, STRIPE_PUBLISHABLE_KEY, USE_HTML5_MODE, LANGUAGE) {
+    console.log(LANGUAGE);
+    var lang = window.localStorage.lang ||LANGUAGE|| 'zh-cn';
+    $translateProvider.preferredLanguage(lang);
```

Comment:
Is there any chance we can ditch the language constants. I think angular translate will do this automatically (https://angular-translate.github.io/docs/#/guide/09_language-negotiation) via: `$translateProvider
  .translations('en', { /* ... */ })
  .translations('de', { /* ... */ })
  .determinePreferredLanguage();`